dao/oracle.go

```diff
@@ -0,0 +1,91 @@
+package dao
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/companieshouse/chs.go/log"
+	"github.com/companieshouse/emergency-auth-code-api/config"
+	"github.com/companieshouse/emergency-auth-code-api/models"
+
+	_ "github.com/godror/godror"
```

Comment:
Our code review standards state that blank imports which are imported only for their side effects should only be used in the main package.
Would this be possible?
https://github.com/golang/go/wiki/CodeReviewComments#import-blank

Reply:
Yeah that's possible. It's just there to initiate the godror driver, doesn't matter where it sits. Makes more sense if it's in main.go so I'll move it there